Classes_and_module/superheros.rb

```diff
@@ -0,0 +1,59 @@
+module Superpower
```

Comment:
Indent with 2 spaces, throughout the code.

Reply:
Changed indentation with 2 spaces throughout the code